test/unit/specs/create-map.spec.js

```diff
@@ -78,6 +78,12 @@ describe('Creating Route Map', function () {
     }).toThrowError(/"path" is required/)
   })
 
+  it('in development, throws if component is null or undefined', function () {
+    process.env.NODE_ENV = 'development'
+    maps = createRouteMap([{ path: '/' }])
+    expect(console.warn).toHaveBeenCalled()
+  })
+
```

Comment:
I would create an spec similar to this one, but to test that this is not affecting production env, with `not. toHaveBeenCalled` 👍